Assets/Scripts/Utils/Saves/SavedHarvestable.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Interactables;
+
+namespace Utils.Saves
+{
+    [System.Serializable]
+    public class SavedHarvestable
+    {
+        public Harvestable harvestable;
+        public bool isRespawning;
+        public double respawnTimePassed;
```

Comment:
All those fields could be made private with [SerializedField] for encapsulation-purposes, but if you don't change it I would be sad but I'll still approve it:)

Reply:
They have to be public so that they correctly serialize to Json...

Reply:
As far as i know, [SerializedField] should do that

Reply:
> Generate a JSON representation of the public fields of an object.

from the Unity Documentation for JsonUtility.ToJson

Reply:
therefore, [SerializeField] does not help when saving to json, the fields have to be public

Reply:
Shouldn't public fields be capitalized, if they're not used in the inspector? Just to keep the consistency...

Reply:
Since they could technically be used in the inspector, I think we should keep it like it ist currently 🤔 But it depends 😄

Reply:
as long as rider does not say anything about capitalization, it will not be capitalized.